docs/api/content-tracing.md

```diff
@@ -78,18 +82,18 @@ Stop recording on all processes.
 
 Child processes typically are caching trace data and only rarely flush and send
 trace data back to the main process. That is because it may be an expensive
-operation to send the trace data over IPC, and we would like to avoid much
-runtime overhead of tracing. So, to end tracing, we must asynchronously ask all
+operation to send the trace data over IPC, and we would like to avoid runtime
+overhead of tracing. So, to end tracing, we must asynchronously ask all
```

Comment:
What do you think about removing the "we" language here?

Something perhaps like:

```
This helps to minimize the runtime overhead of tracing since sending trace data over IPC can be an expensive operation.
```